pkgs/games/nsnake/default.nix

```
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, pkgs, lib}:
```

```suggestion
{ stdenv, fetchFromGitHub, pkgs, lib }:
```
Just a little whitespace typo.

---

Fixed